### PR TITLE
doc(hubble): add docker-compose to start with server

### PR DESCRIPTION
**Pull request by aroundabout**
## Purpose of the PR

- update the way to quick start `server` with `hubble`

## Main Changes

- update the way to quick start `server` with `hubble`

## Verifying these changes

- [x] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [x]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [x]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

File: hugegraph-hubble/README.md [hunk 1]
```diff
@@ -15,7 +15,26 @@ hugegraph-hubble is a graph management and analysis platform that provides featu
...
+        - 18080:8080
+
+    hubble:
+        image: hugegraph/hubble
+        container_name: hubble
+        ports:
+        - 8088:8088
+    ```
+
+Then we should follow the [hubble doc](https://hugegraph.apache.org/cn/docs/quickstart/hugegraph-hubble/#3%E5%B9%B3%E5%8F%B0%E4%BD%BF%E7%94%A8%E6%B5%81%E7%A8%8B) to create the graph.
```
Comment (imbajin) [inline hugegraph-hubble/README.md:37]: use en doc link to avoid the long encode

**Review by imbajin — COMMENTED** [review]
Also update the instrinstruction in root (enhance the desc)

image

File: hugegraph-hubble/README.md [hunk 2]
```diff
@@ -15,7 +15,26 @@ hugegraph-hubble is a graph management and analysis platform that provides featu
...
 ## Quick Start
 
-We can use `docker run -itd --name=hubble -p 8088:8088 hugegraph/hubble` to quickly start [hubble](https://hub.docker.com/r/hugegraph/hubble).
+1. We can use `docker run -itd --name=hubble -p 8088:8088 hugegraph/hubble` to quickly start [hubble](https://hub.docker.com/r/hugegraph/hubble).
+2. We can use the `docker-compose.yml` to start `hubble` with `hugegraph-server`:
+
+    ```
+    version: '3'
+    services:
+    graph_hubble:
```
Comment (imbajin) [inline hugegraph-hubble/README.md:24]: `graph_hubble` seems a little confused?

and shall we add the `preload` param here (comment it or not by default)

File: hugegraph-hubble/README.md (hunk 1 above)
Comment (aroundabout) [reply hugegraph-hubble/README.md]: done

**Review by imbajin — APPROVED** [review]
lgtm

**Review by simon824 — APPROVED** [review]

**Pull request merged**